src/rsz/src/RepairSetup.cc

```diff
@@ -1183,6 +1183,12 @@ Slack RepairSetup::getInstanceSlack(Instance* inst)
         const Slack pin_slack = sta_->vertexSlack(vertex, max_);
         worst_slack = std::min(worst_slack, pin_slack);
       }
+      // Bidi pins may have different slacks at split pins
+      Vertex* d_vertex = graph_->pinDrvrVertex(pin);
```

Comment:
There's no reason to look at pinLoadVertex if we are interested in outputs. I believe the vertex is guaranteed to exist if the graph exists so all we need to do is:
```
  InstancePinIterator* pin_iter = network_->pinIterator(inst);
  while (pin_iter->hasNext()) {
    Pin* pin = pin_iter->next();
    if (network_->direction(pin)->isAnyOutput()) {
      Vertex* vertex = graph_->pinDrvrVertex(pin);
      const Slack pin_slack = sta_->vertexSlack(vertex, max_);
      worst_slack = std::min(worst_slack, pin_slack);
    }
  }
  ```